.travis.yml

```diff
@@ -10,22 +10,41 @@ sudo: false
 addons:
     apt:
         packages:
-            - graphviz
             - texlive-latex-extra
             - dvipng
 
 env:
     global:
-        - MAIN_CMD='python setup.py'
-        - CONDA_CHANNELS='http://ssb.stsci.edu/conda-dev'
-        - CONDA_DEPENDENCIES='pytest jwst sphinx'
-        - CONDA_JWST_DEPENDENCIES='pytest jwst sphinx'
-        - PIP_DEPENDENCIES=''
+        - CONDA_CHANNELS='http://ssb.stsci.edu/astroconda-dev'
```

Comment:
Why the switch from conda-dev to astroconda-dev? Is conda-dev going away?

Reply:
From an administrative point of view we found it easier to write scripts around `astroconda-dev` than `conda-dev`. The original `conda-dev` name isn't going anywhere for historical reasons so you're free to keep using it.

```sh
channel_public="http://ssb.stsci.edu/astroconda"
channel_dev="${channel_public}-dev"
```